minichain/rpc.py

```
@@ -0,0 +1,87 @@
+                address = params[0]
+                account = self.chain.state.get_account(address)
+                result = account["balance"] if account else 0
+            elif method == "mc_sendTransaction":
+                if not params:
+                    raise ValueError("Missing transaction payload")
+                tx_data = params[0]
+                tx = Transaction.from_dict(tx_data)
+                if not tx.verify():
+                    raise ValueError("Invalid signature")
+                if self.mempool.add_transaction(tx):
+                    asyncio.create_task(self.network.broadcast_transaction(tx))
```

Ruff (0.15.15)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)